website/src/components/mainPage/ReadyForAnything/ReadyForAnything.module.scss

```diff
@@ -0,0 +1,138 @@
+
+.title {
+
+    font-size: xx-large;
+    text-align: center;
+
+}
+
+.majorContainer {
+    width: 100%;
+    height: 70vh;
+    margin-top: 100px;
+}
+
+.center {
+    position: relative;
+    margin: 0 30%;
+    top: 10%;
+    width: 40%;
+    text-align: center;
+
+}
+
+.apiItem {
+    height: 100px;
+    width: 100px;
+
+    background-size: contain;
+    background-position: center;
+    background-repeat: no-repeat;
+
+    margin: 20px;
+}
+
+.staticApiItensElement {
+    display: none;
+
+    flex-wrap: wrap;
+    justify-content: center;
+}
+
+.movingApiItensElement .apiItem {
```

Comment:
Can we always have this static please, the orbiting thing feels unnecessary and distracting.